src/modules/showMealDetails.js

```diff
@@ -0,0 +1,61 @@
+const BASE_URL = 'https://www.themealdb.com/api/json/v1/1';
+
+const getMealsDetails = async (idMeal) => {
+  const response = await fetch(`${BASE_URL}/lookup.php?i=${idMeal}`);
+  const meals = await response.json();
+  return meals;
+};
+
+const updateMealDetailsUI = ({ meals }) => {
+  const mealDetailContainer = document.querySelector('.meal-detail-containerr');
+  const [{
+    idMeal,
+    strMeal,
+    strMealThumb,
+    strIngredient1,
+    strIngredient2,
+    strIngredient3,
+    strIngredient4,
+  }] = meals;
+
+  const mealDetails = `
+<img class="meal-image" src="${strMealThumb}" alt="">
+    <h3 class="meal-title">Meal Name: ${strMeal}</h3>
+    <h4 class="meal-title">Ingredients</h4>
+    <div class='ingredients'>
+    <div class="ingredient-pair">
+    <p>${strIngredient1}</p>
+    <p>${strIngredient2}</p>
+    </div>
+    <div class="ingredient-pair">
+      <p>${strIngredient3}</p>
+      <p>${strIngredient4}</p>
+      </div>
+      </div>
+      <input type="hidden"class='item_id' name="item_id" value="${idMeal}">
+  </div>
+`;
+
```

Comment:
- [ ] To ensure that we follow the design template as close as possible, I strongly suggest that you make the design that i requested in the below images.

- [ ] Also, Kindly keep an eye on the font sizes and line heights so that your design looks nice and attractive.

- [ ] Also, kindly give some border-radius and border shadow to the image, it will look attractive.

![image](https://user-images.githubusercontent.com/109666020/203817472-d58b7112-cf0a-49fd-a7e7-ef2c0ae6ff7f.png)